README.md

```diff
@@ -217,13 +248,13 @@ Make sure your build is green before you contribute
 your pull request. You will need to have [Ruby](https://www.ruby-lang.org/en/) 2.3+ and
```

Comment:
Should we raise a minimum Ruby version here as well?
```suggestion
your pull request. You will need to have [Ruby](https://www.ruby-lang.org/en/) 2.7+ and
```

Reply:
Default version on MacOs is 2.6 and it doesn't work on it, it was required to install upper one

Reply:
Fixed